src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -7611,7 +7612,9 @@ abstract public class clx
                 new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractHasBody, Line = 8, Column = 36 },
                 new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractHasBody, Line = 8, Column = 57 },
                 new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractHasBody, Line = 9, Column = 49 },
-                new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractHasBody, Line = 9, Column = 57 });
+                new ErrorDescription { Code = (int)ErrorCode.ERR_AbstractHasBody, Line = 9, Column = 57 },
```

Comment:
Do you mean that all Column properties will be on the same indentation (add additional space before column)? But it will be reformatted by Visual Studio (Ctrl+E, Ctrl+D). So, the next contributor will see changes in git (diff will contain only space changes).